cds/modules/theme/templates/cds_theme/trackingcode.html

```diff
@@ -1,14 +1,18 @@
 {%- if config.THEME_PIWIK_ID %}
+<!-- Piwik -->
 <script type="text/javascript">
-var _paq = _paq || [];
-_paq.push(["trackPageView"]);
-_paq.push(["enableLinkTracking"]);
-(function() {
-  var u=(("https:" == document.location.protocol) ? "https" : "http") + "://piwik.web.cern.ch/";
-  _paq.push(["setTrackerUrl", u+"piwik.php"]);
-  _paq.push(["setSiteId", "{{config.THEME_PIWIK_ID}}"]);
-  var d=document, g=d.createElement("script"), s=d.getElementsByTagName("script")[0]; g.type="text/javascript";
-  g.defer=true; g.async=true; g.src=u+"piwik.js"; s.parentNode.insertBefore(g,s);
-})();
+  var _paq = _paq || [];
+  _paq.push(['setDomains', ['*.videos.cern.ch']]);
```

Comment:
@drjova I think we can keep the domain name like this no?

Reply:
IMHO we don't plan to have any sub-subdomains but it'll work in all cases, so 👍